common/governments/hfe_gov_discovery.txt

```diff
@@ -1,12 +1,135 @@
 # discovery
 
-hfe_gov_discovery = {
+hfe_gov_discovery = { # Elective/Consensus
 	ruler_title = hfe_discovery_captain
 	leader_class = admiral
 	possible = {
-		has_valid_civic = hfe_civic_discovery_habitat
+		OR = {
+			has_origin = hfe_civic_discovery_habitat
+			has_country_flag = hfe_discovery
+			has_country_flag = discovery_men
+		}
+		OR = {
+			has_authority = auth_democratic
+			has_authority = auth_oligarchic
+		}
+		NOT = {
+			has_authority = auth_dictatorial
+			has_authority = auth_imperial
+		}
+		NOT = {
+			has_ethic = ethic_gestalt_consciousness
+		}
+	}
+	election_candidates = {
+		modifier = {
+			add = -20
+			leader_class = governor
+		}
+		modifier = {
+			add = 100
+			leader_class = ruler
+		}
+		modifier = {
+			add = 50
+			leader_class = admiral
+		}
+		modifier = {
+			add = 20
+			leader_class = scientist
+		}
+	}
+	weight = {
+		base = 10000
+	}
+}
+
+hfe_gov_discovery_imperial = {
+	ruler_title = hfe_discovery_emperor
+	ruler_title_female = hfe_discovery_empress
+	heir_title = hfe_discovery_admiral_of_the_fleet
+	heir_title_female = hfe_discovery_admiraless_of_the_fleet
+	use_regnal_names = no
+	dynastic_last_names = yes
+	leader_class = admiral
+	possible = {
+		OR = {
+			has_origin = hfe_civic_discovery_habitat
+			has_country_flag = hfe_discovery
+			has_country_flag = discovery_men
+		}
+		has_authority = auth_imperial
+		NOT = {
```

Comment:
Above trigger already cover it, country cant have 2 auth

Reply:
Okay, done